src/variorum/variorum.c

```diff
@@ -125,36 +160,43 @@ int variorum_print_topology(void)
     int err = 0;
     int i;
     int hyperthreading = 0;
+    hwloc_topology_t topo;
+
+    hwloc_topology_init(&topo);
+    hwloc_topology_load(topo);
 
     err = variorum_enter(__FILE__, __FUNCTION__, __LINE__);
     if (err)
     {
         return -1;
     }
 
+    fprintf(stdout, "=================\n");
     fprintf(stdout, "Platform Topology\n");
-    fprintf(stdout, "  Hostname:             %s\n", g_platform.hostname);
-    fprintf(stdout, "  Num Sockets:          %d\n", g_platform.num_sockets);
+    fprintf(stdout, "=================\n");
+    fprintf(stdout, "  Hostname            : %s\n", g_platform.hostname);
+    fprintf(stdout, "  Num Sockets         : %d\n", g_platform.num_sockets);
     fprintf(stdout, "  Num Cores per Socket: %d\n", g_platform.num_cores_per_socket);
     fprintf(stdout, "  Num Threads per Core: %d\n", g_platform.num_threads_per_core);
     hyperthreading = (g_platform.num_threads_per_core == 1) ? 0 : 1;
     if (hyperthreading == 1)
```

Comment:
This condition could just be "g_platform.num_threads_per_core == 1" but if 'hyperthreading' is being used to improve readability, it's ok to leave it as it is.